administration/src/AuthProvider.tsx

```diff
@@ -24,7 +24,7 @@ const removeToken = () => window.localStorage.removeItem(LOCAL_STORAGE_KEY)
 
 const extractTokenPayload = (token: string): TokenPayload => {
   const payload: { exp: number; adminId: number } = JSON.parse(atob(token.split('.')[1]))
-  const expiry = new Date(payload.exp * 1000) // exp is in seconds, not milliseconds
+  const expiry = new Date(payload.exp * 3600) // exp is in seconds, not milliseconds.
```

Comment:
I feel like this is not the right fix. The "*1000" simply converts to milliseconds.

Reply:
Ok that thats probably true so the payload.exp should be the right one :)

Reply:
Yes, the expiry is set in the backend somewhere.

Reply:
found it and its already 1hour :)